tests/tests_pytorch/accelerators/test_hpu.py

```diff
@@ -323,3 +325,93 @@ def test_hpu_device_stats_monitor(tmpdir):
     ]
     for f in fields:
         assert any(f in h for h in hpu_stats.keys())
+
+
+@RunIf(hpu=True)
+def test_hpu_datamodule():
```

Comment:
let's also add a test where it works

Reply:
But it only seems to support ImageNet dataset, right?

Reply:
1.5 release has imagenet support only. Also, there is an associated test in example